src/views/v2/recent-past.php

```diff
@@ -39,25 +39,25 @@
 		data-view-breakpoint-pointer="<?php echo esc_attr( $breakpoint_pointer ); ?>"
 	<?php endif; ?>
 >
-	<div class="tribe-common-l-container tribe-events-l-container">
+	<div class="recent-past-events-container">
 		<?php $this->template( 'components/loader', [ 'text' => __( 'Loading...', 'the-events-calendar' ) ] ); ?>
 
-		<?php $this->template( 'components/json-ld-data' ); ?>
+<!--		<?php /*$this->template( 'components/json-ld-data' ); */?>
 
-		<?php $this->template( 'components/data' ); ?>
+		<?php /*$this->template( 'components/data' ); */?>
 
-		<?php $this->template( 'components/before' ); ?>
+		<?php /*$this->template( 'components/before' ); */?>
 
-		<header <?php tribe_classes( $header_classes ); ?>>
-			<?php $this->template( 'components/messages' ); ?>
+		<header <?php /*tribe_classes( $header_classes ); */?>>
+			<?php /*$this->template( 'components/messages' ); */?>
 
-			<?php $this->template( 'components/breadcrumbs' ); ?>
+			<?php /*$this->template( 'components/breadcrumbs' ); */?>
 
-			<?php $this->template( 'components/events-bar' ); ?>
+			<?php /*$this->template( 'components/events-bar' ); */?>
 
-			<?php $this->template( 'recent-past/top-bar' ); ?>
+			<?php /*$this->template( 'recent-past/top-bar' ); */?>
```

Comment:
* :no_entry_sign: Expected 1 space before closing PHP tag; 0 found
* :warning: This comment is 67% valid code; is this commented out code?

<sub>hash: 5c2447a4f8c7ee1a4d318ae979993c88</sub>